apps/api/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "strictNullChecks": true,
```

Comment:
Ideally, I think that we should have it defined in the root `tsconfig.base.json`, but I'm not sure if all packages use it... we should only redefine it in the tests like you did

Reply:
I first started as defining it for everything, unfortuently it was to much errors for me to handle. So I focused in this PR on the critical services. We will have to refactor it in the future to enable globally. Wdyt?